app/src/main/res/values/strings.xml

```diff
@@ -656,40 +658,41 @@
     <string name="field_value4">value4</string>
     <string name="profile_metadata">Profile metadata</string>
     <string name="available_mastodon_2_4_later">(available in Mastodon 2.4 or later)</string>
-    <string name="confirm_boost_private_from">Boost private status from %1$s ? It\'s shown to all followers.</string>
-    <string name="boost_private_toot_not_allowed">You can\'t boost private toot by another person.</string>
+    <string name="confirm_boost_private_from">Boost private status from %1$s, shown to all followers?</string>
+    <string name="boost_private_toot_not_allowed">You can\'t boost a private toot by another person.</string>
     <string name="update_push_subscription">Update push subscription (Mastodon 2.4 or later)</string>
-    <string name="pseudo_account_not_supported">Pseudo account can\'t use notifications.</string>
-    <string name="instance_does_not_support_push_api">Instance version %1$s too old. that does not support push subscription.</string>
-    <string name="missing_fcm_device_id">Can\'t prepare FCM device ID. please retry later.</string>
-    <string name="missing_install_id">Can\'t prepare install ID. please retry later.</string>
-    <string name="token_exported">Due to exporting application data or restoring from backup, this access token may be used by other devices as well. Because access tokens that are not used by other devices are required to use push notifications, we recommend updating the access token.</string>
+    <string name="pseudo_account_not_supported">Pseudo accounts can\'t use notifications.</string>
+    <string name="instance_does_not_support_push_api">The instance version %1$s too old. It does not support push.</string>

```

Comment:
please use "push subscription".
old instances have no "push subscription" , but it may have push function for web browser (that can't be used for third app.)